docs/en/compute/Presto-Iceberg.md

```diff
@@ -0,0 +1,107 @@
+---
+layout: global
+title: Querying Iceberg Tables with Presto and Alluxio
+nickname: Presto-Iceberg
+group: Compute Integrations
+priority: 2
+---
+
+Presto has introduced support for [Iceberg tables](https://iceberg.apache.org/) in version 0.256.
+
+This document describes how to use Presto to query Iceberg tables through Alluxio.
+
+* Table of Contents
+{:toc}
+
+In order to use Presto to query an Iceberg table, make sure you have set up Presto, Hive metastore 
+and Alluxio. If you haven't done so, please refer to the 
+[guide]({{ '/en/compute/Presto.html' | relativize_url }}) on general Presto installation 
+and configuration. Most of that guide apply for Iceberg workflows as well, and this document 
+covers the specific instructions for working with Iceberg tables.
+
+Before proceeding, make sure you have a working setup of Presto, Hive metastore and Alluxio, 
+and Presto can access data through Alluxio's filesystem interface. 
+The [guide]({{ '/en/compute/Presto.html' | relativize_url }}) mentions two approaches that 
+Presto can access data through Alluxio: via the filesystem interface, and via the catalog service. 
+Alluxio has not added support for Iceberg in the catalog service yet, so for Iceberg tables Presto 
+needs to access them through Alluxio's filesystem interface.
+
+## Install Alluxio client jar to Presto Iceberg connector
+
+Copy the Alluxio client jar located at `{{site.ALLUXIO_CLIENT_JAR_PATH}}` into Presto Iceberg 
+connector's directory located at `${PRESTO_HOME}/plugin/iceberg/`. Then restart the Presto server:
+
+```console
+$ ${PRESTO_HOME}/bin/launcher restart
+```
+
+Also note that the same client jar file needs to be copied to Hive's `lib` directory.
```

Comment:
Just wanna clarify, why do we need to put the alluxio jar to hive's lib? is it for hive metastore or hive?  Is this step mandatory?

Reply:
What I really meant is that the jar needs to be on the classpath of Hive. I've always copied the jar to that directory, so I went with that. I changed it to align with the instructions from the [Hive article](https://docs.alluxio.io/ee/user/stable/en/compute/Hive.html#basic-setup).

> is it for hive metastore or hive?

For Hive Metastore.

> Is this step mandatory

From what I have exprienced without that jar, Presto will fail to create a new schema with location set to an Alluxio URI. The error is `java.io.IOException No FileSystem for scheme: alluxio`.

Reply:
I see, thank you for clarifying!